cmd/flags/advancedFeatures.go

```diff
@@ -0,0 +1,36 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+)
+
+type AdvancedFeatures []string
+
+const (
+	Htmx              string = "htmx"
+	GoProjectWorkflow string = "githubaction"
+)
+
+var AllowedAdvancedFeatures = []string{string(Htmx), string(GoProjectWorkflow)}
+
+func (f AdvancedFeatures) String() string {
+	return strings.Join(f, ",")
+}
+
+func (f *AdvancedFeatures) Type() string {
+	return "AdvancedFeatures"
+}
+
+func (f *AdvancedFeatures) Set(value string) error {
+	// Contains isn't available in 1.20 yet
+	// if AllowedProjectTypes.Contains(value) {
+	for _, advancedFeature := range AllowedAdvancedFeatures {
+		if advancedFeature == value {
+			*f = append(*f, advancedFeature)
+			return nil
+		}
+	}
+
+	return fmt.Errorf("Advanced Feature to use. Allowed values: %s", strings.Join(AllowedAdvancedFeatures, ", "))
```

Comment:
Go will yell at you if you have error strings that are capitalized. 🤷‍♂️ 
```suggestion
	return fmt.Errorf("advanced feature allowed values: %s", strings.Join(AllowedAdvancedFeatures, ", "))
```

Reply:
Commited!